README.md

```diff
@@ -31,6 +31,8 @@ Activate the app:
 occ app:enable nextbackup
 ```
 
+The app will automatically start running from this point onward without any additional configuration. This requires correctly setup [background jobs](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/background_jobs_configuration.html?highlight=cron), which are needed for a correctly working Nextcloud installation anyway.
```

Comment:
Can you please also mention ownCloud, the app can be used by both systems.

Reply:
I've changed this section a little bit with the latest commit (f054e0000b5017540a142204a6344163f97b1fe0). Hopefully this is good enough now.